<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aulas por Professor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelProfessor">
     <property name="text">
      <string>Professor:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboProfessor">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDisciplina">
     <property name="text">
      <string>Disciplina:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboDisciplina">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnConsultar">
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableResultados">
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>